model/task_lifecycle_test.go

```diff
@@ -3723,6 +3723,16 @@ func TestDisplayTaskUpdates(t *testing.T) {
 		},
 	}
 	assert.NoError(dt2.Insert())
+	dt3 := task.Task{
+		Id:          "dt2",
+		DisplayOnly: true,
+		Status:      evergreen.TaskUndispatched,
+		Activated:   false,
+		ExecutionTasks: []string{
+			"task11",
```

Comment:
Did you mean to add `task12` to the execution tasks? (I don't think it's currently being used anywhere)